setup.cfg

```diff
@@ -38,6 +38,9 @@ install_requires =
     numpy>=1.10.0
     scipy>=1.5.4
     matplotlib>=3.0.0
+    pandas>1.0.0
+    polars>=0.15.0
```

Comment:
do we need pandas and polars? I think the less dependencies the better

Reply:
for pandas export both pandas and pyarrow are required.
so a potential user who wants to export the polars dataframe to pandas would be confused about the error of missing pyarrow and would need to install it manually.

you're right, the less dependencies the better, but right now I tend to think that most users who work with dataframes in python would want to use pandas and then they get this missing module error.

I also would like to add this export function to a cell in a tutorial notebook to show this functionality to the polars-uninitiated. so at least for the `docs` extras pandas would be required.

As discussed in slack, let's keep it there and see if it becomes an issue in the future.